frontend/public/components/environment.jsx

```diff
@@ -228,29 +276,47 @@ export const EnvironmentPage = connect(stateToProps)(
     render() {
       const {errorMessage, success, inProgress, currentEnvVars, stale, configMaps, secrets} = this.state;
       const {rawEnvData, readOnly, obj} = this.props;
+      const isNotBuildObject = _.isArray(rawEnvData);
```

Comment:
I would rename this to something like `isContainerArray` since that's the assumption below.

Reply:
I think I was possessed by yoda when naming this var :)